app/assemblers/assemble_tasks.rb

```diff
@@ -0,0 +1,45 @@
+class AssembleTasks
+  include Mandate
+
+  initialize_with :params
+
+  def call
+    SerializePaginatedCollection.(
+      tasks,
+      serializer: SerializeTasks,
+      meta: {
+        unscoped_total: unscoped_total
+      }
+    )
+  end
+
+  memoize
+  def unscoped_total
+    ::Github::Task::Search.(
+      actions: params[:actions],
+      knowledge: params[:knowledge],
+      areas: params[:areas],
+      sizes: params[:sizes],
+      types: params[:types],
+      repo_url: params[:repo_url],
+      track_id: params[:track_id],
```

Comment:
Could you DRY this up into a helper method pls, so we don't accidently add it to one and not the other.

Reply:
Hmm, I'm confused about this in general actually.